plugins/package-managers/node/src/main/kotlin/Scope.kt

```diff
@@ -19,11 +19,21 @@
 
 package org.ossreviewtoolkit.plugins.packagemanagers.node
 
+import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.config.Excludes
+import org.ossreviewtoolkit.model.utils.DependencyGraphBuilder
 
 internal enum class Scope(val descriptor: String) {
     DEPENDENCIES("dependencies"),
     DEV_DEPENDENCIES("devDependencies");
 
     fun isExcluded(excludes: Excludes): Boolean = excludes.isScopeExcluded(descriptor)
 }
+
+internal fun <D> DependencyGraphBuilder<D>.addDependencies(
+    projectId: Identifier,
+    scope: Scope,
+    dependencies: Collection<D>
+) = addDependencies(projectId, scope.descriptor, dependencies)
```

Comment:
Personally, I'm not (yet) convinced about the usefulness for this overload, as the only difference seems to be that callers can pass `scope` instead of `scope.descriptor`, so I'm having a preference to just do that for clarity.

Reply:
I have to admit that I still do not really see the value (partly because it adds more code than it saves typing), but I won't object.

Reply:
I see. I ended up removing the function again.